### ISSUE TITLE
wrong controlURL and eventSubURL in lib/services/MusicServices.js

### ISSUE THREAD

**Issue by mistaTT** [bencevans/node-sonos#58]
The controlURL and eventSubURL are wrong. For music services the /MediaRenderer/ is not needed, the correct urls are:

`this.controlURL = '/MusicServices/Control';`
`this.eventSubURL = '/MusicServices/Event';`
##

**Comment by stephen** [comment]
PR welcome here. Are you actively using stuff from `services/`? I think the general notion was that it's not really maintained and should/shouldn't be ripped out. Would be interested to hear what you're doing with it.

**Comment by mistaTT** [comment]
I'm using this library and node-webkit to write a SONOS controller. Wenn i called the function ListAvailableServices i got the error "405 method not allowed", so i tried to change the URL from "/MediaRenderer/MusicServices/Control" to /MusicServices/Control and got the correct response with the list of available services. When there is a better way to get the active used music services from a SONOS device, then i would like to try it.

**Comment by ebaauw** [comment]
I encountered the same issue, not just with MediaRenderer, but also for DeviceProperties and ZoneGroupTopology.  Probably some more services as well, but I'm not using these.

I'm new to Sonos and was quite happy to find these services.  I found out by now that they can also be found in the /xml/device_description.xml of your Sonos Player.

I'm integrating Sonos in my Apple Homekit setup by creating a homebridge plug-in.  My ultimate goal is to switch on (or off) automatically  the (Philips Hue) lights and (Sonos) speaker in a room when I enter (or exit) that room.  On enter, I want the room to join automatically another room when that's already playing music; on exit I want the room to leave the group.

I'm new to GitHub (this is actually my first post), so the homebridge plug-in is not on GitHub (yet).
